File: src/main/java/com/timecheck/policy/WorkPolicy.java
package com.timecheck.policy;

import java.time.LocalTime;

public final class WorkPolicy {

    public static final int STD_WORK = 480;
    public static final int STD_HALF = 240;

    public static final int BREAK_BASE = 60;
    public static final int BREAK_OVER = 0;
    public static final int REST_EXTRA = 60;

    public static final LocalTime STD_START = LocalTime.of(9, 0);
    public static final LocalTime STD_END = LocalTime.of(18, 0);
    public static final LocalTime CORE_START = LocalTime.of(10, 0);
    public static final LocalTime CORE_END = LocalTime.of(16, 0);
    public static final LocalTime LUNCH_START = LocalTime.of(11, 30);
    public static final LocalTime LUNCH_END = LocalTime.of(12, 30);

    public static final LocalTime OT_SPLIT = LocalTime.of(22, 0);

    private WorkPolicy() {
    }
}

File: frontend/src/utils/workPolicy.ts
/** Java {@link com.timecheck.policy.WorkPolicy} 와 동일 */
export const WorkPolicy = {
  STD_WORK: 480,
  STD_HALF: 240,
  HALF_DAY_BOUNDARY: { hour: 14, minute: 0 },
  HALF_DAY_HHMM: "14:00",
  BREAK_BASE: 60,
  BREAK_MAIN: 0,
  BREAK_OVER: 60,
  STD_START: { hour: 9, minute: 0 },
  STD_END: { hour: 18, minute: 0 },
  CORE_START: { hour: 10, minute: 0 },
  CORE_END: { hour: 16, minute: 0 },
  LUNCH_START: { hour: 11, minute: 30 },
  LUNCH_END: { hour: 12, minute: 30 },
  OT_SPLIT: { hour: 22, minute: 0 }
} as const;
